_pages/404.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Page Not Found"
+excerpt: "Page not found. Your pixels are in another canvas."
+sitemap: false
+permalink: /404.html
+---
+
+Sorry, but the page you were trying to view does not exist --- perhaps you can try searching for it below.
+
+<script>
+  var GOOG_FIXURL_LANG = 'en';
```

Comment:
Technically we should return it in the language requested, but given the documentation is completely in english... we might have bigger details if 'en' is not a usable language.